app/helpers/ss/editor_helper.rb

```diff
@@ -150,4 +146,69 @@ def html_editor_tinymce(elem, opts = {})
 
   def html_editor_markdown(elem, opts = {})
   end
+
+  def site_ckeditor_editor_options(opts = {})
+    return opts if @cur_site.nil?
+    if @cur_node
+      color_button = @cur_node.color_button || @cur_site.color_button
+      editor_css = @cur_node.editor_css || @cur_site.editor_css
+      editor_css_path = @cur_node.editor_css_path || @cur_site.editor_css_path
+    else
+      color_button = @cur_site.color_button
+      editor_css = @cur_site.editor_css
+      editor_css_path = @cur_site.editor_css_path
+    end
+
+    if color_button == 'enabled'
+      opts[:extraPlugins] ||= ['colorbutton']
+      opts[:removePlugins] ||= []
+      opts[:removePlugins] -= ['colorbutton']
+    elsif color_button == 'disabled'
+      opts[:removePlugins] ||= ['colorbutton']
+    end
+
+    if editor_css == 'enabled'
+      opts[:contentsCss] = editor_css_path
+    elsif editor_css == 'disabled'
+      opts.delete(:contentsCss)
+    end
+
+    opts
+  end
+
+  def site_tinymce_editor_options(opts = {})
```

Comment:
__[rubocop]__
* Perceived complexity for site_tinymce_editor_options is too high. [16/15]

<!-- Comment by [SideCI](https://sideci.com) -->